.eslintignore

```diff
@@ -4,15 +4,3 @@ apps/gipy/pkg/gps.js
 
 # Needs to be ignored because it includes broken JS
 apps/health/chart.min.js
-
-# TypeScript
-apps/btadv/
-apps/clkinfostopw/
-apps/ctrlpad/
-apps/drained/
-apps/folderlaunch/
-apps/popconlaunch/
-apps/rep/
-apps/widChargingStatus/
-apps/widbtstates/
-apps/widhid/
```

Comment:
Can we run the linter on ts files and tweak this to ignore the generated js files?
```suggestion
apps/btadv/app.js
apps/clkinfostopw/clkinfo.js
apps/ctrlpad/main.js
apps/drained/app.js
apps/drained/boot.js
apps/drained/settings.js
apps/folderlaunch/app.js
apps/folderlaunch/configLoad.js
apps/folderlaunch/settings.js
apps/folderlaunch/types.d.js
apps/popconlaunch/boot.js
apps/popconlaunch/settings.js
apps/rep/app.js
apps/rep/settings.js
apps/widChargingStatus/widget.js
apps/widbtstates/widget.js
apps/widhid/wid.js
```

Reply:
This seems like a big maintenance burden but if you don't mind it, that is fine with me :)
An alternative could be to make a plugin that checks to see if there is a pair of ts and js files with the same name, and if true, it instructs the parser to skip the js file.

Reply:
Yes I like the sound of that plugin - leave it with me